app/views/admin/questions/_question.html.slim

```diff
@@ -1,20 +1,21 @@
 -@questions.each do |question|
   tr.info
     td
-      =link_to admin_question_path(question)
+      = link_to admin_question_path(question)
         = question.id
     td
-      =link_to admin_topic_path(question.topic_id)
+      = link_to admin_topic_path(question.topic_id)
```

Comment:
so you see - you spend lot of time for comments and blaming that you can't do this in another way
instead of learning how partials should work
in fact I resolved this issue - just tired continue unnecessary discussion